packages/admin/cms-admin/src/generator/future/generator.ts

```diff
@@ -58,14 +58,15 @@ export type GeneratorConfig = FormConfig<any> | GridConfig<any> | TabsConfig;
 
 export type GeneratorReturn = { code: string; gqlDocuments: Record<string, string> };
 
-export async function runFutureGenerate() {
+export async function runFutureGenerate(specificFile?: string) {
     const schema = await loadSchema("./schema.gql", {
         loaders: [new GraphQLFileLoader()],
     });
     const gqlIntrospection = introspectionFromSchema(schema);
 
     const files = await glob("src/**/*.cometGen.ts");
     for (const file of files) {
+        if (specificFile && file !== specificFile) continue;
```

Comment:
Instead of globbing all *.cometGen.ts files in L67, why not create the files array from specific files?